<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Expected text Sapphire matches actual test Sapphire]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="48316" started-at="2014-06-11T11:33:42Z" finished-at="2014-06-11T11:34:31Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="48316" started-at="2014-06-11T11:33:42Z" finished-at="2014-06-11T11:34:31Z">
      <class name="com.sapphire.admin.Admin_Create_Product">
        <test-method status="PASS" signature="doLogIn()[pri:0, instance:com.sapphire.admin.Admin_Create_Product@1be4777]" name="doLogIn" is-config="true" duration-ms="15015" started-at="2014-06-11T17:03:42Z" finished-at="2014-06-11T17:03:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogIn -->
        <test-method status="FAIL" signature="TestMethodCreateProduct()[pri:0, instance:com.sapphire.admin.Admin_Create_Product@1be4777]" name="TestMethodCreateProduct" duration-ms="23765" started-at="2014-06-11T17:03:57Z" finished-at="2014-06-11T17:04:21Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error in creating product]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error in creating product
	at org.testng.Assert.fail(Assert.java:94)
	at com.sapphire.admin.Admin_Create_Product.TestMethodCreateProduct(Admin_Create_Product.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Expected text Sapphire matches actual test Sapphire]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestMethodCreateProduct -->
        <test-method status="PASS" signature="doLogOut()[pri:0, instance:com.sapphire.admin.Admin_Create_Product@1be4777]" name="doLogOut" is-config="true" duration-ms="9525" started-at="2014-06-11T17:04:21Z" finished-at="2014-06-11T17:04:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogOut -->
      </class> <!-- com.sapphire.admin.Admin_Create_Product -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
